APMrover2/config.h

```diff
@@ -170,4 +170,7 @@
 #else
  #define DEVO_TELEM_ENABLED ENABLED
 #endif
+#ifndef OSD_ENABLED
```

Comment:
OSD_ENABLED == ENABLED

Reply:
I think here should be ifndef, not comparation.

Reply:
ifndef is also used in ArduPlane/config.h